package.json

```diff
@@ -35,6 +35,7 @@
     "@commitlint/cli": "^17.4.0",
     "@commitlint/config-conventional": "^17.4.0",
     "@playwright/test": "^1.29.1",
+    "@types/blocksuite__global-types": "workspace:*",
```

Comment:
Is there any documentation about this `__` syntax, or is it just a naming convention?

Reply:
No, just study from @Jack-Works!

Reply:
Refs: https://github.com/DimensionDev/Maskbook/blob/fdd4a551d468244ce0a7252c47efc7f5639c8b49/tsconfig.json#L29

Reply:
It's documented in https://github.com/DefinitelyTyped/DefinitelyTyped